higher-order-functions/min-max-sorted.md

```diff
@@ -0,0 +1,320 @@
+# Passing Functions as Arguments: Min, Max, and Sort
+
+## Learning Goals
+
+* Demonstrate how to pass a function as an argument
+* Describe how to use the key parameter in the `sorted` function to sort a list of dictionaries by a specific key
+* Describe how to use the key parameter in the `min` function to find the minimum in a list of dictionaries by a specific key
```

Comment:
I had a similar thought. I was wondering if it would it be worth showing an example with lists (like sorting by title length) to explicitly show data structures other than dictionaries can provide custom comparison keys.